modules/apps/commerce/commerce-order-content-web/src/main/java/com/liferay/commerce/order/content/web/internal/frontend/data/set/view/table/PreviewCommerceOrderItemTableFDSView.java

```diff
@@ -32,7 +32,7 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 			_fdsTableSchemaBuilderFactory.create();
 
 		return fdsTableSchemaBuilder.add(
-			"rowNumber"
+			"rowNumber", "N°"
```

Comment:
no, you should use a property from Language.properties, and since it's not there you should add this:
row-n=Row &#8470;

see also figma for reference:
<img width="1398" alt="Screenshot 2024-04-23 at 10 16 23" src="https://github.com/liferay-commerce/liferay-portal/assets/580733/038aa94c-bdfc-4698-8baa-b23dc5ea289e">